staging/src/k8s.io/component-helpers/node/util/sysctl/namespace.go

```diff
@@ -25,24 +25,24 @@ type Namespace string
 
 const (
 	// the Linux IPC namespace
-	ipcNamespace = Namespace("ipc")
+	IPCNamespace = Namespace("ipc")
```

Comment:
Constants are usually named in caps.  `Namespace("IPC")`, `Namespace("Net")` - do these string literals need to be lower-case for some reason?

Reply:
Updated.